testbeam_analysis/gui/analysis_widgets.py

```diff
@@ -603,7 +648,7 @@ def _init_tabs(self):
                         tmp_setup[s_key] = [self.setup[s_key][i]]  # FIXME: Does not work properly without list
                     else:
                         tmp_setup[s_key] = self.setup[s_key][i]
-                elif isinstance(self.setup[s_key], int) or isinstance(self.setup[s_key], str):
+                else:  # isinstance(self.setup[s_key], int) or isinstance(self.setup[s_key], str):
```

Comment:
If not used anymore, remove.

Reply:
...maybe a bad idea, what if if-statement is not True? Else-statement is certainly missing or comment why all other cases are not treated.